types/tx.go

```diff
@@ -73,14 +73,19 @@ func (txs Txs) SplitIntoShares() NamespacedShares {
 		}
 		rawDatas[i] = rawData
 	}
-	shares := splitContiguous(consts.TxNamespaceID, rawDatas)
-	return shares
+
+	w := NewContiguousShareWriter(consts.TxNamespaceID)
+	for _, tx := range rawDatas {
+		w.Write(tx)
+	}
+
+	return w.Export()
 }
 
 // ToSliceOfBytes converts a Txs to slice of byte slices.
 //
 // NOTE: This method should become obsolete once Txs is switched to [][]byte.
-// ref: #2603 https://github.com/tendermint/tendermint/issues/2603
+// ref: #2603
```

Comment:
Why remove the explicit link to Tendermint repo? Should instead remove the `#2603`.

Reply:
not sure, I added it back in, but that issue is *very* old and out of date as there are not plans to switch that type to [][]byte

Reply:
this comment is removed upstream as well, so we shouldn't have to worry about after upgrading